docs/content-media-type-requires-encoding.md

```diff
@@ -0,0 +1,39 @@
+---
+title: \`contentMediaType\` should pair with \`contentEncoding\`
```

Comment:
I don't think this is a good rule. It's perfectly valid to have `contentMediaType` without `contentEncoding`. According to the spec,

> If this keyword is absent, but `contentMediaType` is present, this indicates that the encoding is the identity encoding, meaning that no transformation was needed in order to represent the content in a UTF-8 string.

For example, if `"contentMediaType": "application/json"`, I would not expect `contentEncoding` to be included. In fact it's not clear to me there is a valid value for `contentEncoding` for this case, but I'm not entirely sure about that.

This could be a reasonable rule if it were defined to be smart enough to check for media types that represent binary data (like `image/png`) and require `contentEncoding` for those media types.